stable/xray/values.yaml

```diff
@@ -267,7 +267,9 @@ common:
 # For setting up external services, must pass the connection URL for them
 global:
   mongoUrl:
-  postgresqlUrl:
+  postgresql:
```

Comment:
For not breaking compatibility, need to also support old `postgresqlUrl`. Or existing users' deployments will break.